examples/echo-world/build.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+
+# INSTALL:
+#
+#   brew install docker docker-buildx
+#   mkdir $HOME/.docker/cli-plugins
+#   ln -s $(which docker-buildx) $HOME/.docker/cli-plugins
+#
+#   docker buildx create --name mybuilder --use
+#
+#   this script
+#
+#   docker buildx stop mybuilder
+#   docker buildx rm mybuilder
+
+export DOCKER_HOST=${DOCKER_HOST:-unix://$HOME/.lima/docker/sock/docker.sock}
+
+# get a version - this may be overkill
+if [ -z "$VERSION" ] ; then
+  [ -f VERSION ] || touch VERSION
+  # get value from the file
+  VERSION=$(< VERSION)
+#((VERSION++))
+  let "VERSION=${VERSION-0}+1"
+fi
+
+# may want to point to file's directory not pwd
+# APP=${APP:-${PWD##*/}}
+APP=${APP:-echo-world}
+
+echo "building '$APP' VERSION '${VERSION}'"
+
+# send to docker (no platform?)
+docker buildx build . --load                                    --tag kbrock/${APP}:${VERSION} --tag kbrock/${APP}:latest
```

Comment:
My only issue with this is that `docker buildx` isn't available on most linux distributions' package repositories.  I can't `apt install docker` and run `docker buildx` for example.

It looks like buildah has added support for multiarch builds over in podman https://github.com/containers/buildah/issues/1590

Reply:
This is how we do it for pods: https://github.com/ManageIQ/manageiq-pods/blob/master/bin/build#L57-L99.

For the most part docker build and podman build are basically the same, and if needed we can call out specific podman commands, but even the ones there in manageiq-pods wouldn't be needed here.

Reply:
So, yeah, I agree with @agrare in that we don't need docker buildx.  Additionally I don't think we need the DOCKER_HOST (at least not in the script - my machine works directly without it and I think most other's machines do too)

The question is really whether we need multiarch (for which docker manifest is more than enough), or if linux/amd64 is enough, and whether or not we need the VERSION stuff.

Reply:
We should be able to use what we have https://github.com/ManageIQ/workflows-examples/pull/18

Reply:
agreed, I'd be +1 for an extra container in the workflows-examples repo